services/task-svc/internal/ward/ward.go

```diff
@@ -12,13 +12,13 @@ import (
 	"hwutil"
 	pbhelpers "proto_helpers/task_svc/v1"
 	"task-svc/internal/bed"
-	bedmodels "task-svc/internal/bed/models"
+	bedModels "task-svc/internal/bed/models"
 	"task-svc/internal/patient"
-	roommodels "task-svc/internal/room/models"
+	roomModel "task-svc/internal/room/models"
 	"task-svc/internal/task"
-	templatemodels "task-svc/internal/task-template/models"
+	templateModel "task-svc/internal/task-template/models"
 	"task-svc/internal/ward/models"
-	wardmodels "task-svc/internal/ward/models"
+	wardModel "task-svc/internal/ward/models"
```

Comment:
Should be plural since the files could contain more than one model later on:
e.g. `roomModels`